compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -2059,36 +2059,31 @@ OMR::CodeGenerator::compute64BitMagicValues(
 
    // Cache some common denominators and their magic values.  The key values in this
    // array MUST be in numerically increasing order for the binary search to work.
-   //
-   // The table is composed of 32-bit values because the compiler seems to have a problem
-   // statically initializing it with int64_t constant values.
```

Comment:
I wrote that comment in 2000 because the Microsoft Windows compiler at the time wasn't initializing this table correctly with 64-bit constants.  I'll just assume, 19 years later, that this problem is fixed in current MSVC's...

Reply:
Wow, great to know the reason. Thanks!